.eslintrc-react-test

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@divmain -- You can ticket out an "`.eslintrc-editor`" work if you like! Just don't name it `.eslintrc` as that's special and will interfere with other checks ;)